README.rst

```diff
@@ -70,14 +70,13 @@ A collection is where records are stored.
 
 .. code-block:: python
 
-    # Once the bucket handy, use it to handle collections or groups.
-    collection = bucket.create_collection('receipts')
+    client.create_collection('receipts', bucket='payments')
 
     # Or get an existing one.
-    collection = bucket.get_collection('receipts')
+    client.get_collection('receipts', bucket='payments')
```

Comment:
Actually, yeah, let's do it this way. If people don't want it then they don't use it, but it's a cool feature to have :)